submissions/Nemooochka/friends-app/js/app.js

```diff
@@ -0,0 +1,35 @@
+import {Cards} from "./cards.js";
+import {Url} from "./url.js";
+import {Filters} from "./filters.js";
+
+const cardsWrapBlock = document.querySelector('.wrap-cards');
+
+const urlData = 'https://randomuser.me/api/?results=20';
+let users;
+let filter;
+
+
+(async () => {
```

Comment:
Why did you use IIFE here?

Reply:
To run this code immediately. Or can assign it to a variable and call later, but the idea is the same.

Reply:
Maybe it's not a bad pattern, but using just calling function would be more clear.